sormas-base/pom.xml

```diff
@@ -1141,8 +1163,9 @@
 	<repositories>
 		<repository>
 			<id>central</id>
```

Comment:
I'm not sure if this breaks somewhere, for example for `sormas-app`. It should be tested, but I do see another approach

Reply:
sormas-app has this dependency through the api module - for higher versions of spring it is a problem with android api 24 - but with this version i have tested and is working fine